### PR TITLE
Add unit test for #121: AddressList method fluency

### PR DESCRIPTION
**Pull request by braindawg**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | yes

### Description

This PR provides a failing unit test to demonstrate Issue #121.

### CODE REVIEW

File: test/AddressListTest.php [hunk 1]
```diff
@@ -116,6 +117,18 @@ public function testCanAddManyAddressesAtOnce()
         $this->assertTrue($this->list->has('announce@example.com'));
     }
 
+    public function testCanAddFromStringFluently()
+    {
+        try {
+            $this->list->addFromString('test_fromstring_fluency1@example.com')
+                ->addFromString('test_fromstring_fluency2@example.com');
+        } catch (Throwable $e) {
```
Comment (Ocramius) [inline test/AddressListTest.php]: This `catch` is not necessary

**Review by Ocramius — CHANGES_REQUESTED** [review]
Let's target 2.12.x also here, since it's a bugfix

File: test/AddressListTest.php (hunk 1 above)
Comment (braindawg) [reply test/AddressListTest.php]: Took the cue on that from a couple other unit tests in AddressListTest.php that were similarly `catching` their errors. Removed mine!

**Comment by Ocramius** [comment]
Thanks @braindawg!

**Pull request merged**